docs/docs/configuration/systask.md

```diff
@@ -569,7 +569,7 @@ Each iteration of loop over task will be scheduled as taskRefname__iteration. It
 Do while task output number of iterations with iteration as key and value as number of iterations. Each iteration's output will be stored as, iteration as key and loopover task's output as value
 Taskname which contains arithmetic operator must not be used in loopCondition. Any of loopOver task can be reference outside do while task same way other tasks are referenced.
 To reference specific iteration's output, ```$.LoopTask['iteration]['first_task']```
-Do while task does NOT support domain or isolation group execution.
+Do while task does NOT support domain or isolation group execution. Nesting of DO_WHILE task is not supported. Loopover task must not be reused in neither workflow nor another DO_WHILE task.
```

Comment:
> Loopover task must not be reused in neither workflow nor another DO_WHILE task.

Please provide more details about this. Does this mean that if a task is used as a loopover task, it can be used again within the same workflow?

Reply:
No. It must not be reused within the same workflow once the task is used as loopOver task.